app/controllers/ActivationController.scala

```diff
@@ -22,15 +22,15 @@ import connectors.ApplicationClient
 import connectors.UserManagementClient.{ TokenEmailPairInvalidException, TokenExpiredException }
 import helpers.NotificationType._
 import security.Roles._
-import security.SignInUtils
+import security.SignInService
 
 import scala.concurrent.Future
 
-object ActivationController extends ActivationController {
+object ActivationController extends ActivationController(ApplicationClient) {
   val http = CSRHttp
```

Comment:
Actually this started when I was going to create a test for a controller in faststream-frontend. Then I realised there was not any (it comes, as you know, from fasttrack-frontend). So I investigated other projects to find out why and I realised there were test for controllers in other projects like fasttrack-admin-frontend and fasttrack (although is a different kind of project as it is backend).
So, in those two projects we mostly use the non cake pattern, so then I told Olly we should standardize on something and only change it if there is a good reason. So we concluded to follow fasttrack-admin-frontend way, where we get rid of the cake pattern for dependencies that are of a different nature.
So you can see we "inject" ApplicationClient which we can say is in a different layer but we "with" SignInService which is a kind of "Sub" Controllers.
The main reason to prefer the "injection" was that we think it is easier to test, in particular using mockito, it seems with cake pattern we have to create a comprehensive test double.